.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
   - sudo add-apt-repository -y "ppa:ubuntugis/ppa"
   - sudo add-apt-repository -y "ppa:cran/travis"
   - travis_apt_get_update
-  - sudo apt-get install -y --no-install-recommends build-essential gcc g++ libblas-dev liblapack-dev libncurses5-dev libreadline-dev libjpeg-dev libpcre3-dev libpng-dev zlib1g-dev libbz2-dev liblzma-dev libicu-dev cdbs qpdf texinfo libssh2-1-dev gfortran jq
+  - sudo apt-get install -y --no-install-recommends build-essential gcc g++ libblas-dev liblapack-dev libncurses5-dev libreadline-dev libjpeg-dev libpcre3-dev libpng-dev zlib1g-dev libbz2-dev liblzma-dev libicu-dev cdbs qpdf texinfo libssh2-1-dev gfortran jq python3.7 python3-pip
```

Comment:
it looks like the highest version of python available for Ubuntu Xenial is python3.5 (https://packages.ubuntu.com/xenial/python3.5). Would this work for you?

Reply:
Yes, I just put in a new commit.  How were you able to tell that was the problem?

Reply:
in the logs of the failed travis build: https://travis-ci.com/carpentries/amy-feeds/builds/128943671#L245